control-plane/csi-controller/src/client.rs

```diff
@@ -320,11 +323,19 @@ impl MayastorApiClient {
         size: u64,
         allowed_nodes: &[String],
         preferred_nodes: &[String],
+        inclusive_pool_topology: &HashMap<String, String>,
     ) -> Result<Volume, ApiClientError> {
-        let topology = Topology::explicit(ExplicitTopology::new(
-            allowed_nodes.to_vec(),
-            preferred_nodes.to_vec(),
-        ));
+        let exclusive_label_topology: HashMap<String, String> = HashMap::new();
+        let topology = Topology::new_all(
+            Some(NodeTopology::explicit(ExplicitNodeTopology::new(
+                allowed_nodes.to_vec(),
+                preferred_nodes.to_vec(),
+            ))),
+            Some(PoolTopology::labelled(LabelledTopology::new_all(
+                exclusive_label_topology.to_owned(),
```

Comment:
Nit: You could just do `HashMap::new()` here and get rid of `exclusive_label_topology` altogether.

Reply:
Corrected. Thanks.